src/Button/index.tsx

```diff
@@ -0,0 +1,64 @@
+import type { FC, PropsWithChildren } from "react";
+
+import "./index.scss";
+import clsx from "clsx";
+
+export interface ButtonProps {
+  /**
+   * The button's size.
+   */
+  size?: "sm" | "lg";
+
+  /**
+   * The button's color scheme.
+   */
+  color?: "black" | "white" | "dark-blue" | "red" | "blue-a";
+
+  /**
+   * Whether the button is rounded.
+   */
+  rounded?: boolean;
+
+  /**
+   * Aria label for the button.
+   */
+  ariaLabel?: string;
```

Comment:
While we're here, let's expose an optional `id?: string` prop for HTML IDs. This will allow us to use `htmlFor="..."` in more complex scenarios if the element can't be labeled by its children but we want the label text present.